tests/test_performance.py

```diff
@@ -72,7 +72,7 @@ class ExplicitHarCaptureTest(HarCaptureTestBase):
     any environment setting. Note that this will only work if the `TestClass` is
     decorated, not the `test_case`.
     """
-    @expectedFailure
+    @skip  # Expected Failure. TODO.
```

Comment:
might be nice to annotate a "reason" for each skip, rather than using a bare skip.

such as:

@skip("expected failure")